en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+            <tr class="even">
+                <td>User self registration SMS OTP expiry time</td>
+                <td>
+                    <ul>
+                        <li>User self registration verification link expiry time.</li>
+                        <li>The <b>default and minimum</b> value is <code>1</code> minute.
+                    <ul>
+                </td>
+            </tr>
+        </tbody>
+        </table>
+        
+## Configuring self-registration consent purposes
+    
+For more details see, 
+[Configuring self-registration consent purposes](../.
+./learn/self-registration-and-account-confirmation/#configuring-self-registration-consent-purposes).
+
+## Try out self-registration  
+
+WSO2 Identity Server provides the functionality to confirm the user account internally or externally.
+
+1. **Internal Notification Management** : Manage notification sending an account confirmation 
+by the Identity Server.
+    
+2. **External Notification Management** : Manage notification by an external notification management 
+mechanism and confirm the registration flow to the WSO2 Identity Server.
+
+!!! note
+    To configure external notification management, `disable` the property `Enable Notification 
+    Internally Management` using the self registration configurations.
+    
+    ![external-notification-management-config](
+    ../assets/img/learn/self-registration/external-notification-management-config.png)
+    
+### **Self Registration with Internal Notification Management** 
+
+
+1. Use the following command to create a user self-registration request. 
+    
+    ``` tab="If the preferred channel is EMAIL" 
+    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username": "kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/identity/preferredChannel","value": "EMAIL"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "947721584558"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+    ```
+    
+    ``` tab="If the Preferred channel is SMS" 
+    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username":"kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/identity/preferredChannel","value": "SMS"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "94774747551"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+    ```
+    
+    !!! note
+        for more information, see 
```

Comment:
"for" should start with a capital "F" since its the beginning of a sentence.

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549